src/renderer/src/components/Messages/VoiceInput.tsx

```
@@ -69,73 +67,126 @@ export default function VoiceInput({
+
+      console.log('[SEND] sendAudios succeeded:', result);
+      setAudioChunks([]);
+      dispatch(resetAudioStatus());
+    } catch (err) {
+      console.error('[SEND] sendAudios failed:', err);
+      // Ensure UI is not stuck
+      dispatch(resetAudioStatus());
+    }
   };
 
   const handleStartRecording = async () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)